presto-geospatial-toolkit/pom.xml

```diff
@@ -20,6 +20,11 @@
             <artifactId>esri-geometry-api</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.openjdk.jol</groupId>
+            <artifactId>jol-core</artifactId>
```

Comment:
@findepi JOL is used in GeometryMemorySizeUtilsPackageWorkaround, hence, limiting the scope of the dependency to just "test" seems wrong. 

I submitted a PR to add OGCGeometry::estimateMemorySize() API to ESRI library and it got accepted. If all is well, I should be able to upgrade to a new version of the library and delete GeometryMemorySizeUtilsPackageWorkaround soon.

https://github.com/Esri/geometry-api-java/issues/156

Reply:
Apologies. I trusted too much the IDE to check whether project compiles after changing `pom.xml`. Bad, since the IDE squashes compile's and test's classpaths together.